src/pages/NewOrder.js

```diff
@@ -0,0 +1,122 @@
+import { useNavigate} from "react-router-dom"; 
+import { useId, useState } from "react";
+import styled from "styled-components";
+
+import getDate from "../utils/getDate";
+
+
+export default function NewOrder({onHandleSubmit}){
+    const navigate = useNavigate(); 
+    const [orderType, setOrderType] = useState("Eintrag");
+    
+    const orderId= useId();
```

Comment:
`useId()` from react is not for generating ids for your data. It's for creating unique ids for html elements on both server and client side. If you need unique ids for your data, consider using a package like nanoid.